app/assets/stylesheets/application/components/checkin.scss

```diff
@@ -0,0 +1,67 @@
+.checkin {
+  .media {
+    .media-body {
+      width: 100%;
+
+      .top {
+        height: 17px;
+        line-height: 17px;
+
+        .name {
+          margin-right: 5px;
+        }
+
+        .pull-right {
+          font-size: 12px;
+
+          i.modified {
+            color: $color-gray-dark-light;
+            margin-right: 5px;
+          }
+
+          .created-at {
+            a {
+              color: $color-gray-dark-light;
+            }
+          }
+        }
+      }
+
+      .middle {
+        margin: 7px 0 20px 0;
+      }
+
+      .bottom {
+        font-size: 12px;
+
+        .top {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5